x/dex/types/keys.go

```diff
@@ -51,8 +51,14 @@ func TwapPrefix(contractAddr string) []byte {
 	return append(KeyPrefix(TwapKey), KeyPrefix(contractAddr)...)
 }
 
-func PricePrefix(contractAddr string) []byte {
-	return append(KeyPrefix(PriceKey), KeyPrefix(contractAddr)...)
+func PricePrefix(contractAddr string, priceDenom string, assetDenom string) []byte {
```

Comment:
can we add an example for how the prefix is added in a comment, and also a test for this

Reply:
added a comment and a test